### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by AxelSoler**
This is my first Mycroverse proyect
I created a web with the title 'hello world' and it is in blue.
I used linters for the first time.
Updated README file
Sorry if my English is not the best

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+My first Mycroverse proyect
+Build in html and CSS
```
Comment (AnselemOdims) [inline README.md]: - [x] Please make sure the Microverse README template is used to create the README. You can find the template [here](https://github.com/microverseinc/readme-template/blob/master/README.md)

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @AxelSoler , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] The Pull Request should have a descriptive title telling you what changes are inside. Kindly ensure that the title of the is descriptive
- [x] The Pull Request should have a summary describing introduced changes in more detail. Please ensure to edit the summary and make it descriptive
- [x] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by AxelSoler** [comment]
thank you
I changed the pull name and description
and updated README file

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @AxelSoler , This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

##Required - 

- [ ] Kindly make sure to add the pull request title as per the project title, for example, You can rename the title to Hello Microverse Project.

 ## Highlights
- Linters passing
- Professional Readme

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AxelSoler** [comment]
hello  @sja-thedude, I didn't understand very well
Do I have to change the name of the repository and the pull request?

**Comment by sja-thedude** [comment]
Hi , @AxelSoler Just change this pull request name to` Hello Microverse Project` and submit it for review. I will approve it.

**Comment by AxelSoler** [comment]
thank you

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @AxelSoler, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**